### PR TITLE
misc(github): add github PR and issue templates

### PR DESCRIPTION
**Pull request by underbyte**
## Feature request

**Summary of the feature request?**

Github supports these [templates](https://blog.github.com/2016-02-17-issue-and-pull-request-templates/) to  help standardize contribution into repository. Whenever I'm filing a ticket/PR having these templates help in the writing process. What submitted here are heavily inspired by the webpack community; more possible format can be found [here](https://github.com/stevemao/github-issue-templates).

**What is the motivation or use case for changing this?**
To help standardize the format for writing PR/issue tickets.

**How is this beneficial to Ligthhouse?**
Help make it easier for people to contribute via PR/issue tickets.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+**What is the current behavior?**
+
+
+**What is the expected behavior?**
+
+
+**Provide the steps to reproduce if this is a bug**
+
+
+**Please mention other relevant information such as the browser version, Node.js version, Lighthouse version, and Operating System.**
```
Comment (patrickhulce) [inline .github/ISSUE_TEMPLATE.md]: Let's borrow from the bug report style below:


```
**Environment Information**:
Affected Channels:
- [ ] CLI
- [ ] Node Module
- [ ] Extension
- [ ] DevTools
Lighthouse version:
Node.js version:
Operating System:
```

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
```
Comment (patrickhulce) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: [From what I understand](https://blog.github.com/2018-01-25-multiple-issue-and-pull-request-templates/) these will only be used if you append `?template=Feature_request.md` to the new issue URL?

If so, I'm not sure we have a use case for the specific issue templates just yet. Maybe we should just start with global issue template.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+**Summary**
+<!-- What kind of change does this PR introduce? -->
+<!-- Is this a bugfix, feature, refactoring, build related change, etc? -->
+
+<!-- Describe the need for this change -->
+
+<!-- Link any documentation or information that would help understand this change -->
+
+**Other information**
```
Comment (patrickhulce) [inline .github/PULL_REQUEST_TEMPLATE.md]: WDYT about changing this into `**Related Issues/PRs**` we frequently end up linking blocking PRs/closing issues/etc

Other seems like something one can always add on their own :)

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+**What is motivation or use case for changing this?**
+
+
+**Please mention other relevant information such as the browser version, Node.js version, Lighthouse version, and Operating System.**
+
+
+<!-- If this is a bug report use this portion of the template and delete the feature request portion -->
+
+**Describe the *bug* issue**
```
Comment (patrickhulce) [inline .github/ISSUE_TEMPLATE.md]: I like this set of questions! WDYT about adding one about relevant existing issues?

```html
<!-- Have you searched for similar open issues? -->
<!-- If yours is a duplicate, consider commenting with additional info or +1'ing instead. -->
Related issues: <!-- #9000 -->
```
or something

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+
+**What is the current behavior?**
+
+
+**What is the expected behavior?**
+
+
+**Provide the steps to reproduce if this is a bug**
```
Comment (patrickhulce) [inline .github/ISSUE_TEMPLATE.md]: we've already established this is the bug part so let's just say `Provide the steps to reproduce`

File: .github/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+<!-- Before creating an issue please make sure you are using the latest version. -->
+
+<!-- If this is a feature request use this portion of the template and delete the bug portion -->
+**Describe the *feature* request**
+
+
+**What is motivation or use case for changing this?**
+
+
+**Please mention other relevant information such as the browser version, Node.js version, Lighthouse version, and Operating System.**
```
Comment (patrickhulce) [inline .github/ISSUE_TEMPLATE.md]: we can probably just ask where they want the feature (CLI/DevTools/Extension/Node Module)

File: .github/ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+<!-- Before creating an issue please make sure you are using the latest version. -->
+
+<!-- If this is a feature request use this portion of the template and delete the bug portion -->
+**Describe the *feature* request**
```
Comment (patrickhulce) [inline .github/ISSUE_TEMPLATE.md]: We also have a process for new audits we can reference https://github.com/GoogleChrome/lighthouse/blob/master/docs/new-audits.md

File: .github/ISSUE_TEMPLATE/Feature_request.md (hunk 2 above)
Comment (underbyte) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: Yes, it does change the URL but you also get this (at least, if I understand it right).

screen shot 2018-05-14 at 8 38 23 pm

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (underbyte) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sounds good to me!

File: .github/ISSUE_TEMPLATE.md (hunk 7 above)
Comment (underbyte) [reply .github/ISSUE_TEMPLATE.md]: went ahead and added this to the feature request template

**Comment by underbyte** [comment]
@patrickhulce Think I got everything. Let me know if anything else is missing.

File: .github/ISSUE_TEMPLATE/Feature_request.md (hunk 2 above)
Comment (patrickhulce) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: BTW where did that screenshot come from?

Comment (underbyte) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: The SS came from webpack repo [LINK](https://github.com/webpack/webpack/issues/new/choose).

Think the global one would be nice to have? Wouldn't hurt to keep it around.

Comment (paulirish) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: Ya let's nuke the global issue template. The specific ones are good 'nuff.

File: .github/ISSUE_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+
+<!-- If this is a bug report use this portion of the template and delete the feature request portion -->
+
+**Describe the *bug* issue**
+
+
+<!-- Have you searched for similar open issues? -->
+<!-- If yours is a duplicate, consider commenting with additional info or +1'ing instead. -->
+Related issues: <!-- #9000 -->
```
[the remark this answers is not in the document]
Comment (underbyte) [reply .github/ISSUE_TEMPLATE.md]: Latest [commit](https://github.com/GoogleChrome/lighthouse/pull/5163/commits/f683162aff53b7513acfaf99a8b21004968449ae), I delete the issue template and added your last two feedback to the bug report.

**Comment by underbyte** [comment]
Hey @patrickhulce, anything left on this PR?

**Pull request merged**